### ISSUE TITLE
Midpoint Review - IA Feedback - Transition Experience, tbd, V1 Transition Experience

### ISSUE THREAD

**Issue by erinrwhite** [department-of-veterans-affairs/va.gov-team#88203]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Thank you for coming through the Collab Cycle!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

### Must

- [x] **Consistent titles** The H1, current breadcrumb, and first segment of the `<title>` tag before `| Veterans Affairs` should all be the same text.

- [x] **Within form flow, open any links in a new tab** In general, whenever folks are in a flow where they might lose progress/saved data if they leave the page, we encourage teams to [open links in a new tab](https://design.va.gov/components/link/#behavior). When opening links in a new tab, also add the text "(opens in a new tab)" at the end of the link. This would apply to these links:
  - "Learn more about the discharge upgrade process" on the Discharge page
  - "Learn more about disability ratings" on the Disability ratings page
  - "Privacy policy" link on the review page

- [x] **Use action link on Getting Started page** The [getting started page](https://www.figma.com/design/DNy6D8ELfTQW1g3EK7yJu5/Mid-Fidelity-Prototype-for-v1?node-id=2683-30695&t=B0LgrBKwLayMP3EF-4) is using a button to take folks into the form flow and should use an action link instead. Rule of thumb, [buttons](https://design.va.gov/components/button/) do things, [links](https://design.va.gov/components/link/action) go places.

- [x] **Remove "back to review" link on results page** Instead of a "back to review" link, the [breadcrumbs](https://design.va.gov/components/breadcrumbs) should be updated (something like VA.gov home > Benefits questionnaire > Your results). 

- [x] **Work with CAIA IA's if possible on breadcrumbs** I encourage you to work with CAIA IA's @jennymayoco @kristinoletmuskat to iron out the breadcrumbs for this entire flow and ensure that other elements of the IA are solid.

- [x] **Address flow question for users who have not yet separated from service:** Our current flow is:
  - Are you in the service?
    - When will you separate?
  - Separation info
  - Discharge info
  
  What about people who have not yet separated or been discharged - people who might be current servicemembers who are curious about their VA benefits once they separate in the future? Especially since we are asking detailed questions about a future separation date on the preceding page, there should be a way for people who have not yet separated/been discharged to move through the flow. Perhaps it's an "I have not yet separated" option on the "Time since separation" question that then allows people to skip the discharge question? 
![Screenshot 2024-07-12 at 10 48 44 AM](https://github.com/user-attachments/assets/315ae978-7b99-45e5-9be2-57feea6d7788) The CAIA IA's could also help with this.

### Should

- [ ] **Results page information hierarchy** The [results page](https://www.figma.com/design/DNy6D8ELfTQW1g3EK7yJu5/Mid-Fidelity-Prototype-for-v1?node-id=2683-30695&t=B0LgrBKwLayMP3EF-4) is the most complex in the app and the one I'd tackle first as you work on your mobile view.

  Think about the flow of information on the page in a linear pattern - one heading, subheading, and element at a time. What should the order of operations be? How might headings add structure to the page content in a way that people who navigate by headings (i.e. screen reader users, people scanning the page) would understand quickly? With that in mind, I'd recommend:

  - Place an H2 higher on the page, above the filters, that encapsulates the content below - your recommended benefits and resources
  - Pull the results out of the accordions
  - Rather than showing benefits folks might not be eligible for in an accordion, add a filter or checkbox for users to "Show benefits I may not qualify for"
 - It's great that the “Showing 2 results...” text will update any time a filter or sort is applied. This is how it should work! Please follow guidance from @briandeconinck about the best way to implement this pattern for screen readers.

- [x] **Include findability task(s) in research** I understand that y'all are talking with OCTO DE leadership including @mnorthuis about entry points for this tool. Mikki and I both recommend talking with users to understand where they might expect to find paths to a tool like this on VA.gov, and also what words they might use to describe or name it.

- [x] **Always use back/continue button pair** On the [goals page](https://www.figma.com/design/DNy6D8ELfTQW1g3EK7yJu5/Mid-Fidelity-Prototype-for-v1?node-id=2683-30694&t=tR3DdKTKv2d5XURm-4), for consistency, inclue back and continue button pair rather than just continue.

### Consider

- [x] **Info flow on goals page** On the [goals page](https://www.figma.com/design/DNy6D8ELfTQW1g3EK7yJu5/Mid-Fidelity-Prototype-for-v1?node-id=2683-30694&t=tR3DdKTKv2d5XURm-4), consider putting the explanatory paragraph above the list of options.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by cora-goldston-bah** [comment]
Hi @erinrwhite and team — thank you for reviewing and providing feedback! We are working through the feedback and updating our prototype.

We have a question about the last "Must" item about response options for those who are current servicemembers. We recognize some current servicemembers have re-enlisted after previously separating from the service, so they are both Veterans and current servicemembers. In these cases, we'd still want to know the person's character of discharge from their previous service period(s), since they may be eligible for additional benefits.

We realize that in our current questionnaire setup, the "time since separation" question would be confusing for servicemembers who are also Veterans. If we add a response option like "I have not yet separated," then these servicemembers will have to choose between that option and the response option that applies to the time since their previous separation.

Here's an idea for a potential solution — what are your thoughts?

- The user sees the question "Are you currently serving in the military?"
    - If they answer No, they go directly to the time since separation question
    - If they answer Yes, they still get the conditionally revealed question about their expected separation date, but then they are taken to another question that asks, "Have you ever completed a previous period of military service?"
        - If they answer No to this question, then they skip the time since separation and character of discharge questions
            - We don't necessarily want to skip the subsequent disability rating question because a current servicemember may have already applied and received a VA disability rating, especially if they're nearing their separation date
        - If they answer Yes, they're taken to the time since separation screen and continue through the questionnaire

We are open to other ideas, so please let us know your thoughts. Thank you!

**Comment by erinrwhite** [comment]
@cora-goldston-bah this sounds like a good solution! Thank you for taking the time to revise this. 

You might also consider, rather than making the expected separation date a conditionally revealed field, having it display on a separate page. It adds another page to the flow but keeps the one-thing-per-page model and potentially keeps people moving through the form quickly. Not required at all, just a thought.

**Comment by cora-goldston-bah** [comment]
@erinrwhite Perfect — thanks so much! We'll implement the flow described above and we'll also consider moving the expected separation date to a separate page.

**Comment by kristinoletmuskat** [comment]
Also @cora-goldston-bah on the breadcrumbs in Erin's MUST feedback (and for [URLs](https://design.va.gov/components/url-standards)), feel free to reference the design system guidance to propose a draft of breadcrumbs/URLs for each page, and I can review it! 

We're in the middle of revising the breadcrumb guidance to be clearer for forms, but in the meantime you can review this example of how breadcrumbs currently work at VA. Basically, throughout the form, the last item in the breadcrumb should be the official plain-language title of the form (which usually matches the H1 on the 'intro' page, but not always!). 

![Screenshot 2024-07-23 at 3 46 58 PM](https://github.com/user-attachments/assets/cbd1e257-5072-4015-b6b3-af9166d83e77)

**Comment by RonnieYoung-UX** [comment]
Hi all! Just wanted to use this ticket to reach back for review on the mocks/sketches we brought to office hours last week. I've attached the images below. ([They're on this figma file](https://www.figma.com/design/DNy6D8ELfTQW1g3EK7yJu5/Mid-Fidelity-Prototype-for-v1?node-id=0-1&t=CZAuGQlDTE9Wi943-1))

We stayed away from using tabs based on the feedback we received. We went with the two column approach like other pages on VA's website. I'm working on the mobile mocks now but wanted to get this in front of the team sooner rather than later. 

Thank you!


![Results Page - Answered All Questions - 31JULY - Expanded dropdown](https://github.com/user-attachments/assets/b277f16a-3d95-427f-a3f6-5221f687f39a)
![Results Page - Answered All Questions-31JULY](https://github.com/user-attachments/assets/e5ca7e9b-ca3a-4501-9b2f-9ffbde1082ba)
